### ISSUE TITLE
 RUNTIME ERROR: undefined external variable: BERT_MODEL_NAME

### ISSUE THREAD

**Issue by Srijha09** [successar/influence_info_repo#1]
I am getting a runtime error while training. 
![image](https://user-images.githubusercontent.com/50697493/198380489-1a30cf0e-3e91-4339-8304-ecc75e275cfa.png)

Can you please help me solve this error?

**Comment by successar** [comment]
Hi,

Can you include BERT_MODEL_NAME="bert-base-uncased" as env var to the command also? If it still gives an error, let me know.

**Comment by Srijha09** [comment]
Thank you, the code works. 

Also, does it take a really long time to run the influence value command using bash?
The code almost ran for 4 hours.

**Comment by successar** [comment]
Yes, I believe the most time consuming part is calculating influence via this https://github.com/successar/instance_attributions_NLP/blob/master/influence_info/influencers/influence_functions.py since it involves calculating hessians vector products.

**Comment by Srijha09** [comment]
Thank you, also just a quick doubt, for the nearest neighbors, once you get the influence_values.npy file, is there a threshold value that is used to remove 500 training data points in order to retrain it?

**Comment by successar** [comment]
No, we just remove the top 500 examples before retraining.

**Comment by Srijha09** [comment]
Thanks, just to confirm, are these top 500 samples randomly selected or is there an inherent ranking based on some criteria?

**Comment by successar** [comment]
It depends on the value of the influence method. So in case of NN, you would remove the top-500 examples with highest NN similarity for a given test point. The value of the influence method is the criteria.
